src/Framework/TestSuite.php

```diff
@@ -131,6 +145,8 @@ class PHPUnit_Framework_TestSuite implements PHPUnit_Framework_Test, PHPUnit_Fra
      */
     public function __construct($theClass = '', $name = '')
     {
+        $this->numTestsRun = 0;
```

Comment:
This assignment is superfluous as the attribute is initialized to this value upon declaration.